vmdb/app/controllers/dashboard_controller.rb

```diff
@@ -303,13 +303,13 @@ def widget_toggle_minmax
         page << javascript_add_class("w_#{w}_minmax", "minbox")
         page << javascript_del_class("w_#{w}_minmax", "maxbox")
         page << javascript_show("dd_w#{w}_box")
-        page << "$('w_#{w}_minmax').title = 'Minimize';"
+        page << "$j('#w_#{w}_minmax').prop('title', 'Minimize');"
         @sb[:dashboards][@sb[:active_db]][:minimized].delete(w)
       else
         page << javascript_add_class("w_#{w}_minmax", "maxbox")
         page << javascript_del_class("w_#{w}_minmax", "minbox")
         page << javascript_hide("dd_w#{w}_box")
-        page << "$('w_#{w}_minmax').title = 'Restore';"
+        page << "$j('#w_#{w}_minmax').prop('title', 'Restore';)"
```

Comment:
@skateman semicolon on this line should be outside parenthesis, this is preventing me from minimizing dashboard widgets.